test/widget_tests/splash_screen_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:movies_app/presentation/view/main_screen.dart';
+import 'package:movies_app/presentation/view/splash_screen.dart';
+import 'package:animated_splash_screen/animated_splash_screen.dart';
+import 'package:mockito/annotations.dart';
+import 'splash_screen_test.mocks.dart';
+
+@GenerateNiceMocks(
+  [
+    MockSpec<NavigatorObserver>(),
+  ],
+)
+void main() {
+  late MockNavigatorObserver mockNavObserver;
+
+  setUp(() {
+    mockNavObserver = MockNavigatorObserver();
+  });
+
+  Widget buildWidget() {
+    return MaterialApp(
+      home: const SplashScreen(
+        key: Key('screenKey'),
+      ),
+      routes: {
+        'SplashScreen': (context) => const SplashScreen(),
+        'MainScreen': (context) => const MainScreen(),
```

Comment:
Add this routes to a route constant file

Reply:
Done.